docker-compose.yml

```diff
@@ -86,7 +86,7 @@ services:
   #
   mock-auth:
     # See: https://github.com/navikt/mock-oauth2-server#standalone-server
-    image: ghcr.io/navikt/mock-oauth2-server:0.4.8
+    image: ghcr.io/navikt/mock-oauth2-server:2.1.1
     volumes:
```

Comment:
I think it will be important to rebuild the image as well, so running
`docker-compose up --detach --build`

Reply:
I deleted the container/image, then ran the `--build` command to ensure I got it updated, I don't know if it was needed but that was what I did.